src/main/java/hudson/plugins/spotinst/cloud/BaseSpotinstCloud.java

```diff
@@ -199,13 +201,57 @@ public String getDisplayName() {
         return this.name;
     }
 
+    //For Jelly use
+    public String getIconAltText() {
+        String retVal;
+        String emptyString      = "";
+        String superIconAltText = this.getClass().getSimpleName().replace("Cloud", "");
+        ;
+        String labelToolTip =
+                StringUtils.isEmpty(labelString) ? emptyString : String.format("%nLabels: %s", labelString);
```

Comment:
use StringUtils.empty instead of emptyString

Reply:
done